Makefile

```
@@ -200,7 +193,11 @@ publish:
 		exit 1; \
 	fi
 	$(MAKE) prepublish
+ifeq ("$(BABEL_8_BREAKING)", "true"))"
```

Looks like there's an unmatched `"` and `)` here

```suggestion
ifeq ("$(BABEL_8_BREAKING)", "true")
```

(I'm not sure about the remaining quotes, https://www.gnu.org/software/make/manual/html_node/Conditional-Example.html omits them.)

---

Thanks 😅

For the other quotes, I just copy-pasted from a few lines above because I know it works.